app/rifcs/rif_cs_generator.rb

```diff
@@ -115,7 +118,12 @@ def build_rif_cs
           wrapper_object.related_websites.each do |related_website|
             xml.relatedInfo type: 'website' do
               xml.identifier related_website, type: 'uri'
-              xml.title ''
+              begin
```

Comment:
Use the wrapper_object to do this - take a look at package_rif_cs_wrapper.rb - that is where this code belongs.